scripts/install.sh

```diff
@@ -4,17 +4,23 @@
 set -e
 
 INPUT_COLOR="\033[1;36m"
-MSG_COLOR="\033[0;31m"
+MSG_COLOR="\033[0;32m"
+ERROR_COLOR="\033[0;31m"
 NO_COLOR="\033[0m"
 
+# Ensure the working directory is the project root directory.
+
+cd "${0%/*}/../"
```

Comment:
Wow what??

Reply:
Go up one directory from the script, basically. This addresses the case where someone would execute the script from an unexpected directory.